mod/wikirate/spec/set/right/contributed_analysis_spec.rb

```diff
@@ -5,11 +5,11 @@
 
     analysis = get_a_sample_analysis
 
-    article = Card[analysis.name].fetch :trait=>:wikirate_article,:new=>{}
+    article = Card[analysis.name].fetch :trait=>:overview,:new=>{}
     article.content = "One of my most productive days was throwing away 1000 lines of code."
     article.save!
 
-    article1 = Card["Apple_Inc+Natural Resource Use"].fetch :trait=>:wikirate_article,:new=>{}
+    article1 = Card["Apple_Inc+Natural Resource Use"].fetch :trait=>:overview,:new=>{}
```

Comment:
Line is too long. [86/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator `=>`.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.